src/yoroi-wallets/utils.ts

```diff
@@ -85,3 +86,23 @@ export const Quantities = {
     return new BigNumber(quantity1).dividedBy(new BigNumber(quantity2)).toString() as Quantity
   },
 }
+
+export const toYoroiAmounts = (utxos: RawUtxo[], defaultAssetId: TokenId): YoroiAmounts => {
+  return utxos.reduce(
+    (amounts: YoroiAmounts, current: RawUtxo) => {
+      amounts[defaultAssetId] = Quantities.sum([amounts[defaultAssetId], `${new BigNumber(current.amount).toNumber()}`])
```

Comment:
what type is `current.amount`?
`current.amount -> BigNumber -> number -> string`
if its already a string, then just use that as is?

Reply:
`Quantity` is `${number}`

Reply:
Fixed thanks!